### PR TITLE
custom network interface

### PR DESCRIPTION
**Pull request by 1allen**

### CODE REVIEW

**Comment by fcojean** [comment]
Thank you for this contribution. I have merged and tagged it. A new docker image will be soon available on docker hub with tag 1.1.0.

**Comment by 1allen** [comment]
glad to hear, Francois! awesome!
thank you!

File: run.sh [hunk 1]
```diff
@@ -36,6 +31,16 @@ if [ -z "$VPN_IPSEC_PSK" ] || [ -z "$VPN_USER_CREDENTIAL_LIST" ]; then
   exit 1
 fi
 
+if [ -z "$VPN_IPSEC_INTERFACE" ]; then
+  VPN_IPSEC_INTERFACE="eth0"
```
Comment (1allen) [inline run.sh:35]: oh, seems I forgot to rename the variable to `VPN_NETWORK_INTERFACE`, sorry

Comment (fcojean) [reply run.sh:35]: yes. FIxed in 1.1.1.

**Pull request merged**